src/main/java/org/folio/marccat/dao/persistence/Authority008Tag.java

```diff
@@ -111,8 +201,8 @@ public char getBilingualUsage() {
     return getAutItm().getBilingualUsage();
   }
 
-  public char getSubjectDescriptor() {
-    return getAutItm().getSubjectDescriptor();
+  public void setBilingualUsage(char c) {
+    getAutItm().setBilingualUsage(c);
```

Comment:
It's clear that there's an expectation that `getAutItm()` will never return null, but I wanted to verify with you that will always be the case.

Reply:
We think that the `autItm` never will be null, but I have checked it.